spec/commands/init_spec.rb

```diff
@@ -64,6 +64,22 @@
     end
   end
 
+  context "when the dir is not writable by the current user", :bundler => ">= 2" do
```

Comment:
This shouldn't be tagged with `:bundler => ">= 2"` because this is not testing any specific Bundler 2 functionality.

Reply:
Is there any other 1.x release planned?

Reply:
Yes but that isn't why i'm saying don't use `:bundler => ">= 2"`. We use that tag when we have to test something specifically for Bundler 2 such as UI changes like new options to commands. The error you're asserting for in your spec won't change between Bundler 1 and Bundler 2.

Reply:
yes

Reply:
oh, alright.  I’ll remove that tag there then and try to make the test pass with Ruby 1.8